spec/system/case_study_spec.rb

```diff
@@ -0,0 +1,41 @@
+RSpec.describe "CaseStudy" do
+  before do
+    content_store_has_example_item("/government/case-studies/get-britain-building-carlisle-park", schema: :case_study)
+    content_store_has_example_item("/government/case-studies/doing-business-in-spain", schema: :case_study, example: "doing-business-in-spain")
+  end
+
+  it_behaves_like "it has meta tags", "case_study", "/government/case-studies/doing-business-in-spain"
+  it_behaves_like "it has meta tags for images", "case_study", "/government/case-studies/doing-business-in-spain"
+
+  context "when visiting a Case Study page" do
+    it "displays the case_study page" do
+      visit "/government/case-studies/get-britain-building-carlisle-park"
+
+      expect(page).to have_title("Get Britain Building: Carlisle Park - Case study - GOV.UK")
+
+      expect(page).to have_css("h1", text: "Get Britain Building: Carlisle Park")
+      expect(page).to have_text("Nearly 400 homes are set to be built on the site of a former tar distillery thanks to Gleeson Homes and HCA investment.")
+
+      expect(page).to have_css(".gem-c-translation-nav")
+    end
+
+    context "when visiting a Withdrawn Case Study page which is also translatable" do
```

Comment:
As mentioned above, the "translatable" part of this context should be extracted into a separate context, along with the expectation.